src/ORM/Connect/MySQLDatabase.php

```diff
@@ -226,25 +226,25 @@ public function searchEngine(
 
         // Generate initial DataLists and base table names
         $lists = array();
-        $baseClasses = array($pageClass => '', $fileClass => '');
```

Comment:
`$baseClasses` wasn't even used. Just delete it instead of fixing it?
We already have $pageTable and $fileTable, so we don't need to do it again here.